src/xrpld/consensus/Consensus.cpp

```diff
@@ -181,6 +181,12 @@ checkConsensus(
         return ConsensusState::MovedOn;
     }
 
+    if (currentAgreeTime > parms.ledgerMAX_CONSENSUS + previousAgreeTime)
```

Comment:
I think that because of this condition here, we are unable to test the change in `DisputedTx.h` - can you engineer timings such that we will test the last `newPosition = false` in `DisputedTx::updateVote` as well ?

Reply:
> I think that because of this condition here, we are unable to test the change in `DisputedTx.h` - can you engineer timings such that we will test the last `newPosition = false` in `DisputedTx::updateVote` as well ?

This has been revised, too.